src/aiodynamo/http/httpx.py

```diff
@@ -20,5 +21,7 @@ async def __call__(self, request: Request) -> Response:
                 content=cast(bytes, request.body),
             )
             return Response(response.status_code, await response.aread())
+        except httpx.TimeoutException:
+            raise asyncio.TimeoutError
```

Comment:
```suggestion
            raise asyncio.TimeoutError()
```

Reply:
Thank you! Fixed and made a commit again.